src/explore-education-statistics-common/src/services/publicationService.ts

```diff
@@ -10,6 +10,23 @@ import { contentApi } from './api';
 
 export type ReleaseApprovalStatus = 'Draft' | 'HigherLevelReview' | 'Approved';
 
+// TODO EES-3517 a guess at the new type for the find stats page.
+export interface PublicationSummaryWithRelease {
+  id: string;
+  legacyPublicationUrl?: string;
+  release?: {
+    id: string;
+    published: string;
+    theme: {
+      title: string;
+    };
```

Comment:
Themes aren't associated to releases. It goes Theme > Topic > Publication > Release.

Would put this on the publication itself:

```ts
export interface PublicationSummaryWithRelease {
  theme: {
    title: string;
  }
}
```